package.json

```diff
@@ -5,34 +5,39 @@
   "author": "Skuid",
   "bugs": "https://github.com/skuid/skuid-sfdx/issues",
   "dependencies": {
-    "@oclif/command": "^1",
-    "@oclif/config": "^1",
-    "@oclif/errors": "^1",
-    "@salesforce/command": "^2",
-    "@salesforce/core": "^2",
+    "@oclif/core": "^1.20.0",
```

Comment:
With this package, I was running into some issues coming from `@oclif/core` and `@salesforce/sf-plugins-core`. Updating to newer versions of both seemed to fix it, and I found [a few threads](https://github.com/forcedotcom/cli/issues/2529) like this with the same error. So I think we should update them to `"@oclif/core": "^2.15.0"` and `"@salesforce/sf-plugins-core": "^3.1.21"` to avoid this.